app/keycloak/protocolMappers.ts

```diff
@@ -82,14 +80,26 @@ export const createPrivacyZoneMapper = async (
       };
     }
 
-    await kcAdminClient.clients.addProtocolMapper(
-      {
-        id: clientId,
-        realm,
-      },
-      config,
-    );
+    if (!mapperId) {
+      await kcAdminClient.clients.addProtocolMapper(
+        {
+          id: clientId,
+          realm,
+        },
+        config,
+      );
+    } else {
+      await kcAdminClient.clients.updateProtocolMapper(
+        {
+          id: clientId,
+          realm,
+          mapperId,
+        },
+        { ...config, id: mapperId },
+      );
+    }
   } catch (err) {
+    console.log('🚀 ~ err:', err);
```

Comment:
Just checking if this is intentionally in, might make sense to log the err here

Reply:
oh thats just left out from my debug